Sources_v3/StreamChatUI/ChatChannel/ChatMessage/ChatMessageQuoteBubbleView.swift

```diff
@@ -0,0 +1,44 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+open class ChatMessageQuoteBubbleView<ExtraData: ExtraDataTypes>: MessageComposerQuoteBubbleView<ExtraData> {
+    public var isParentMessageSentByCurrentUser: Bool? {
+        didSet {
+            updateContentIfNeeded()
+        }
+    }
+    
+    override open func setUpLayout() {
+        super.setUpLayout()
+        
+        // In ChatMessage view is added to superview in `setUpLayout` but all constraint are activated in
+        // `updateContent`
+        // This workaround is preventing ambiguous layout when constraints for this view will be activated
+        // before superview constraints.
+        container.removeFromSuperview()
+        
+        container.preservesSuperviewLayoutMargins = false
+        container.isLayoutMarginsRelativeArrangement = false
+    }
+    
+    override open func updateContent() {
+        super.updateContent()
+    
+        if message == nil {
+            container.removeFromSuperview()
+            return
+        } else if container.superview == nil {
+            embed(container)
+        }
+        
+        if isParentMessageSentByCurrentUser == true {
```

Comment:
Why are we showing incoming and outgoing messages in composer same way (colors and avatar positioning)?

Reply:
Colors are different - you can see it on the screenshot.
It's set inside `ChatMessageQuoteBubbleView`.
Regarding avatar position - there was a message from designer team that avatar should be always on the left.
Let me check if I can find it.

Reply:
Seems like I confused it with different comment so thanks for spotting this 😉

<img width="300" alt="Screenshot 2021-01-15 at 14 16 52" src="https://user-images.githubusercontent.com/10098359/104732277-452b0180-573d-11eb-9bcf-07b3913717d4.png">